app/javascript/modal.js

```diff
@@ -156,11 +156,11 @@ document.addEventListener('DOMContentLoaded', function() {
   for (var i=0;i<confirmableLinks.length;i++) {
     confirmableLink = confirmableLinks[i];
     confirmableLink.addEventListener('click', function(event) {
-      var userVerified = event.target.getAttribute('data-user-verified');
+      var userVerified = confirmableLink.getAttribute('data-user-verified');
```

Comment:
If an `a` tag wraps a couple other elements, one of them is likely to be the `target`. Instead use the closed over `confirmableLink` to ensure we're always talking about the `a` tag here.